src/pages/home/report/ReportActionCompose.js

```diff
@@ -509,7 +533,7 @@ class ReportActionCompose extends React.Component {
         const isComposeDisabled = this.props.isDrawerOpen && this.props.isSmallScreenWidth;
         const isBlockedFromConcierge = ReportUtils.chatIncludesConcierge(this.props.report) && User.isBlockedFromConcierge(this.props.blockedFromConcierge);
         const inputPlaceholder = this.getInputPlaceholder();
-        const hasExceededMaxCommentLength = this.comment.length > CONST.MAX_COMMENT_LENGTH;
+        const hasExceededMaxCommentLength = this.state.exceededCommentLength != null;
```

Comment:
You are comparing `this.state.exceededCommentLength` to `null` here, but AFAICT it's never set to `null`, only `undefined`

Reply:
Since the check is `!=` - `null` and `undefined` are coerced and both are treated as `false` 
If the check was `!==` then we'd have the problem where the value must explicitly not match `null` 

I'm surprised this is not caught by an eslint rule that prevents implicit comparisons 
Usually js community uses only `===` and `!==` to avoid bugs (e.g. if someone refactor this `!= null` to `!== null` it'll stop working)

Reply:
Oh, good catch.

> I'm surprised this is not caught by an eslint rule that prevents implicit comparisons

Agreed

Reply:
Yeah, i'd say it depends on the guidelines of the project. As I understand you, we want explicit `null` and `undefined` checks, hence I am going to change it 👍